modules/apps/commerce/commerce-theme-minium/commerce-theme-minium/src/css/minium/components/minium_frame.scss

```diff
@@ -37,19 +37,8 @@
 	}
 
 	&__content {
-		padding: #{$frame-size} 0 0 $frame-size;
-
-		@media screen and (hover: none) {
-			padding: #{$frame-size} 0 $frame-size 0;
-		}
-
-		.has-control-menu.signed-in & {
-			padding: #{$frame-size + $liferay-topbar-size} 0 0 $frame-size;
-
-			@media screen and (hover: none) {
-				padding: #{$frame-size + $liferay-topbar-size} 0 $frame-size 0;
-			}
-		}
+		padding: calc(#{$frame-size} + var(--control-menu-container-height)) 0 0
+			$frame-size;
```

Comment:
What about the bottom padding on @media screen and (hover: none)?

Reply:
@smottal This looked way over complicated to me and likely incorrect. I tried testing it on mobile to see if anything looked wrong with my changes and it appeared fine. Liferay in general simply uses width based media queries, so a different way of testing for small screen sizes makes things hard. 

We should be able to remove all of those additional checks, because the --control-menu-container-height css variable automatically adjusts for screen size and whether topbars are present.

Reply:
@ethib137 what I can see is that by removing `@media screen and (hover: none)` that removes the left padding by adding it at the bottom I get different results

This image is the current behavior: the padding is at the bottom (probably unnecessary) and no padding on the left and the content is centered on the page
<img width="1573" alt="current" src="https://github.com/liferay-commerce/liferay-portal/assets/117469730/7e6e24f8-1eb4-4aa6-ae60-2ad2d8eeacbc">

This image is using the PR: the padding at the bottom is no set (and this should not be a problem, at least in this page) but there is padding on the left causing the content to be "uncentered"
<img width="1601" alt="pr" src="https://github.com/liferay-commerce/liferay-portal/assets/117469730/4468c241-0506-42f4-9bbc-9fe56024e844">

Do you have the same result?

Reply:
Oh okay... I see now. I'm guessing this is to compensate for a switch to the menu location on mobile. I'll update this PR with a new commit. Thank you.

Reply:
@smottal fixed in a new commit. Please let me know if you see anything else that needs changing. Thanks.